sources/packages/web/src/components/students/StudentApplicationsExtendedSummary.vue

```diff
@@ -21,7 +21,12 @@
           :loading="loading"
           :mobile="isMobile"
           @update:options="paginationAndSortEvent"
+          show-expand
+          v-model:expanded="expandedItems"
         >
+          <template v-slot:loading>
+            <v-skeleton-loader type="table-row@5"></v-skeleton-loader>
+          </template>
           <template #[`item.applicationNumber`]="{ item }">
             {{ item.applicationNumber }}
           </template>
```

Comment:
Aren't we removing the column ? I thought so when I saw the `applicationName` property being removed. 
![image](https://github.com/user-attachments/assets/86e15e4a-fe52-47f1-8ad4-b1994a36c51d)

Reply:
Yes, removed from the headers model, the UI removed the column, and I forgot to removeit  from the Vue template.

Reply:
It is removed now.